src/pattern-library/components/patterns/feedback/PopoverPage.tsx

```diff
@@ -0,0 +1,176 @@
+import { Link } from '../../../../components/navigation';
+import Library from '../../Library';
+
+export default function PopoverPage() {
+  return (
+    <Library.Page
+      title="Popover"
+      intro={
+        <>
+          <p>
+            <code>Popover</code> is a floating component that can be shown
+            next/relative to another anchor element.
+          </p>
+        </>
+      }
+    >
+      <Library.Section
+        title="Popover"
+        intro={
+          <>
+            <p>
+              By default, <code>Popover</code> will be displayed under the
+              anchor element, unless there is not enough space below.
+            </p>
+            <p>
+              It will always be at least as big as the anchor element, but it
+              can grow to fit its content if needed. However, it will never grow
+              or be displayed outside of the viewport.
+            </p>
+            <p>
+              In browsers that support it, <code>Popover</code> uses the{' '}
+              <Link
+                target="_blank"
+                href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/popover"
+              >
+                <code>popover</code>
+              </Link>{' '}
+              attribute and gets toggled via{' '}
+              <Link
+                target="_blank"
+                href="https://developer.mozilla.org/en-US/docs/Web/API/Popover_API"
+              >
+                popover API
+              </Link>
+              . Otherwise, it is rendered as an absolute-positioned element, so
+              it is recommended to wrap it and its anchor element in a
+              relative-positioned container.
```

Comment:
The examples in this page _don't_ do this, so if you modify them to set `asNativePopover={false}` they render incorrectly.